src/Website/Resources/orderconfirmationemail.php

```diff
@@ -6,14 +6,14 @@
 //$mail->SMTPDebug = 3;                               // Enable verbose debug output
 
 $mail->isSMTP();                                      // Set mailer to use SMTP
-$mail->Host = 'mail.smtp2go.com';  // Specify main and backup SMTP servers
+$mail->Host = 'smtp.sendgrid.net';  // Specify main and backup SMTP servers
 $mail->SMTPAuth = true;                               // Enable SMTP authentication
-$mail->Username = 'orders@stuffbuysharks.xyz';                 // SMTP username
-$mail->Password = 'CS3250!!';                           // SMTP password
+$mail->Username = 'apikey';                 // SMTP username // apikey CS3250Key2
+$mail->Password = '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';                           // SMTP password  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
 $mail->SMTPSecure = 'TLS';                            // Enable TLS encryption, `ssl` also accepted
-$mail->Port = 2525;                                    // TCP port to connect to
+$mail->Port = 587;                                    // TCP port to connect to
 
-$mail->setFrom('stuffbysharks@gmail.com', 'Shark Mailer');
+$mail->setFrom('stuffbuysharks@gmail.com', 'Shark Mailer');
```

Comment:
derek demonstrated this working for jesse and I. can confirm the connection works.